unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs

```diff
@@ -360,4 +359,16 @@ private async UniTaskVoid WaitForRequestTimeOutThenHideLoadingFeedback()
         View?.SetLoadingMessagesActive(false);
         View?.SetOldMessagesLoadingActive(false);
     }
+    
+    private void HandleMessageBlockedBySpam(ChatMessage message)
+    {
+        chatHudController.AddChatMessage(new ChatEntryModel
+        {
+            timestamp = (ulong) DateTimeOffset.UtcNow.ToUnixTimeMilliseconds(),
+            bodyText = "You sent too many messages in a short period of time. Please wait and try again later.",
```

Comment:
Is this message already confirmed by producto? Maybe we could indicate the amount of time the user will have to wait.

Reply:
yes it is confirmed!